### ISSUE TITLE
Adjust width and height for multiple columns / rows at once

### ISSUE THREAD

**Issue by tfulge** [davidgohel/flextable#46]
First of all, thank you for making `flextable`. It is an amazing package that is set to help me a great deal in generating a Word report with lots of tables at work.

I am trying to format my pretty wide `flextable` object to fit into a `.docx` file. Adjusting widths and heights for individual columns / rows is easy using the `width` and `height` options, but I am wondering whether it would also possible to apply changes for a number of columns / rows at a time.

Right now, this is my procedure:

    ft <- ft(width, j = ~ variableA, width = 0.6)
    ft <- ft(width, j = ~ variableB, width = 0.6)
    [...]
I am aware I can apply changes to all columns at once by simply typing `ft <- ft(width, width = 0.6)`, but width is supposed to vary by column type.

Ideally, it would be possible to either specify vectors of columns (e.g. `ft <- ft(width, j = ~ c(variableA, variableB), width = 0.6)` and / or specify multiple columns in one just one call to `width` (though I don't know how that could be accomplished given the structure of the syntax as is).

Maybe I am missing an obvious way why this cannot work or even that the solution is already implemented in the package, and I apologize if that should be the case.

**Comment by davidgohel** [comment]
Hi thanks,

Please follow the guidelines written in the issue's template.

**Comment by tfulge** [comment]
I did, and I thought I adhered to the guidelines, seeing this was meant as an enhancement idea.

Unless of course I am mistaken and what I am after is already implemented. I'll scour the documentation once more and head on over to Stack Overflow to frame this as a question. Apologies!

**Comment by davidgohel** [comment]
Oh sorry, it's just that users that provide `sessionInfo()` and a reproducible example are rare so it became an automatic thing ;)

What you are asking is already implemented. You only have to provide a `j` corresponding to multiple columns and provide the `width` with as many values as there are in your columns selection.

**Comment by tfulge** [comment]
Got it, thank you! I was typing in column names, the fact that column numbers can also be specified somehow didn't occur to me, even though that is precisely what I had done for the rows. Sorry for not being more diligent and thank you again for the great package!

**Comment by davidgohel** [comment]
Yes, or `~ colname1 + colname2`, there are many ways to specify columns
